src/main/scala/com/gradle/internal/Env.scala

```diff
@@ -0,0 +1,31 @@
+package com.gradle.internal
+
+import Env.Key
+
+object Env {
```

Comment:
Can we keep these in the `Utils` class and if possible have similar names to the previous functions to make it easier to align functionality with previous Gradle/Maven?